library/xml-model/src/main/java/org/cru/godtools/xml/model/Content.kt

```diff
@@ -37,11 +46,16 @@ abstract class Content : BaseModel {
 
             return when (parser.namespace) {
                 XMLNS_CONTENT -> when (parser.name) {
-                    Paragraph.XML_PARAGRAPH -> Paragraph(parent, parser)
+                    Paragraph.XML_PARAGRAPH -> when (parser.getAttributeValue(null, Paragraph.XML_FALLBACK)
+                        ?.toBoolean()) {
+                        true -> Fallback(parent, parser)
+                        else -> Paragraph(parent, parser)
```

Comment:
So can you explain the difference where we would use Fallback vs. Paragraphe? Is it because either can be a paragraph, but one could have additional context that isn't just text?

Reply:
fallback is meant to be purely render the first child element that is supported. right now we are leveraging paragraphs to provide a mostly backwards compatible mechanism to introduce the fallback concept until apps are upgraded and directly support the `fallback` node directly